components/common/src/ui.rs

```diff
@@ -43,6 +44,169 @@ pub const NONINTERACTIVE_ENVVAR: &str = "HAB_NONINTERACTIVE";
+    pub fn to_str(&self) -> &str {
+        let style = if let Ok(s) = env::var(SYMBOL_STYLE_ENVVAR) {
+            match UISymbolStyle::from_str(&s) {
+                Ok(style) => style,
+                Err(e) => {
+                    eprintln!("{}\nSetting UISymbolStyle to {}",
+                              e,
+                              UISymbolStyle::default());
+                    UISymbolStyle::default()
+                }
+            }
+        } else {
+            UISymbolStyle::default()
+        };
+
+        match style {
+            UISymbolStyle::Ascii => {
+                match *self {
+                    UISymbol::UpArrow => "/^\\",
+                    UISymbol::FingerPoint => "-->",
+                    UISymbol::CheckMark => "[x]",
+                    UISymbol::BoxedCheckMark => "[#]",
+                    UISymbol::Omega => "-->",
+                    UISymbol::BoxedX => "-X-",
+                    UISymbol::RightArrow => "-->",
+                    UISymbol::Cloud => "-->",
+                    UISymbol::DownArrow => "  >",
+                    UISymbol::Elipses => "...",
+                    UISymbol::DottedTriangle => "???",
+                    UISymbol::RightShift => " >>",
+                    UISymbol::Star => "***",
+                    UISymbol::SlashedZero => "  0",
+                    UISymbol::ErrorX => "XXX",
+                }
+            }
+            UISymbolStyle::Limited => {
+                match *self {
+                    UISymbol::UpArrow => "↑",
+                    UISymbol::FingerPoint => "→",
+                    UISymbol::CheckMark => "√",
+                    UISymbol::BoxedCheckMark => "⌂",
+                    UISymbol::Omega => "Ω",
+                    UISymbol::BoxedX => "░",
+                    UISymbol::RightArrow => "→",
+                    UISymbol::Cloud => "⌂",
+                    UISymbol::DownArrow => "↓",
+                    UISymbol::Elipses => "…",
+                    UISymbol::DottedTriangle => "‼",
+                    UISymbol::RightShift => "»",
+                    UISymbol::Star => "≡",
+                    UISymbol::SlashedZero => "Ø",
+                    UISymbol::ErrorX => "XXX",
+                }
+            }
+            UISymbolStyle::Full => {
+                match *self {
+                    UISymbol::UpArrow => "↑",
+                    UISymbol::FingerPoint => "☛",
+                    UISymbol::CheckMark => "√",
+                    UISymbol::BoxedCheckMark => "☑",
+                    UISymbol::Omega => "Ω",
+                    UISymbol::BoxedX => "☒",
+                    UISymbol::RightArrow => "→",
+                    UISymbol::Cloud => "☁",
+                    UISymbol::DownArrow => "↓",
+                    UISymbol::Elipses => "…",
+                    UISymbol::DottedTriangle => "∵",
+                    UISymbol::RightShift => "»",
+                    UISymbol::Star => "★",
+                    UISymbol::SlashedZero => "Ø",
```

Comment:
I wonder if we want a slightly different character (and correspondingly name) here:
∅
EMPTY SET
Unicode: U+2205, UTF-8: E2 88 85

Reply:
Though, based on use, I think a more appropriate name would be `::Warning` and the more appropriate glyph may be
⚠︎
WARNING SIGN
Unicode: U+26A0 U+FE0E, UTF-8: E2 9A A0 EF B8 8E

Reply:
yeah I admit it seemed odd that we were using this character. I don't get the connection with warning. I just didn't want this PR to be about changing the underlying symbols where they already render.

Reply:
I mean, we're already doing that to a point, right? If you want to have another PR that directly follows up on this one, I'm happy with that, but otherwise I worry we won't ever come back to fix this situation.